master/buildbot/steps/package/rpm/mock.py

```diff
@@ -25,16 +25,16 @@
 
 
 class MockStateObserver(logobserver.LogLineObserver):
-    _line_re = re.compile(r'^.*State Changed: (.*)$')
```

Comment:
I imagine this RE was correct at some point. Could you document which version of mock we now support?
I dont think this is worth to support both, but we should document it.

Reply:
Okay, I added it in the docstring. As it should still work, just not update the build step description on mock progress, perhaps it doesn't need documentation in the actual user docs?